soumyo/aws_hosted_zone_all_record_issue_fix (#811)
inspec/inspec-aws: docs/resources/aws_hosted_zones.md

platform: aws
---

# aws_hosted_zones

> Use the `aws_hosted_zones` InSpec audit resource to test the properties of the plural resource of AWS Route53 HostedZone.

Reviewer: ```suggestion
Use the `aws_hosted_zones` InSpec audit resource to test the properties of multiple AWS Route53 hosted zones.
```

> The AWS::Route53::HostedZone creates a new public or private hosted zone.

Reviewer: ```suggestion
The `AWS::Route53::HostedZone` creates a new public or private hosted zone.
```

## Syntax

> ##### Ensure the hosted zones are available

Reviewer: ```suggestion
Ensure the hosted zones are available
```

    describe aws_hosted_zones do
      it { should exist }
    end

#### Parameters

This resource does not expect any parameters.

> For additional information, see the [AWS documentation on AWS Route53 HostedZone](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-route53-hostedzone.html).

Reviewer: ```suggestion
For additional information, see the [AWS documentation on the `AWS::Route53::HostedZone` resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-route53-hostedzone.html).
```
